docs/gds-supported-platform/resizing-persistent-volumes.md

```diff
@@ -0,0 +1,96 @@
+# Resizing persistent volumes
+
+The volumes you created in the beginning may not be big enough for
+your ongoing needs.
+
+As an example, we're going to double the space on the Concourse Worker
+persistent volumes - from 64Gi to 128Gi. Replace
+`gsp-concourse-worker` with the name of the resource you want to
+resize the volume for.
+
+1. Make a change to `charts/gsp-system/values.yml` with your desired
+   sizing, like [in PR 343](https://github.com/alphagov/gsp/pull/343):
+
+```yaml
+...
+persistence:
+  worker:
+    size: 128Gi
+```
+
+1. When merged, the PR will be released to sandbox. Things must pass
+   through sandbox via the full resize process to make sure we haven't
+   broken anything before going to the Verify cluster.
+
+1. Pods in StatefulSets each have a volume. For Kubernetes to be able
```

Comment:
is it StatefulSet, statefulset, or stateful set?

Reply:
The k8s docs refer to StatefulSet pretty consistently, so we'll go with that!